boards/st/nucleo_g431kb/doc/index.rst

```diff
@@ -0,0 +1,189 @@
+.. _nucleo_g431kb_board:
+
+ST Nucleo G431KB
+################
+
+Overview
+********
+
+The Nucleo G431kB board features an ARM Cortex-M4 based STM32G431kB MCU
```

Comment:
Maybe a spell checker played tricks on you? Just noticed this sorry


```suggestion
The Nucleo G431KB board features an ARM Cortex-M4 based STM32G431KB MCU
```

Reply:
The port started with a copy from the Nucleo G431RB board and some find and replace actions